ibrahim/styles.css

```
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+
+/* Undoing defaults */
+body {
+  margin: 0 0 0 0;
+  transition: all 750ms;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+figure {
+  margin: 0;
+}
+
+img {
+  height: auto;
+  width: 100%;
+}
+
+/* ---------- */
+
+body {
+  font-family: "Quicksand", sans-serif;
+  color: #1e1e1e;
+  font-size: 10px;
+  background: linear-gradient(#f0f0f0, #ffffff);
+}
+
+.text {
+  margin: 20px 25px;
+}
+
+h1 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 36px;
+}
+
+h2 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 18px;
+}
+
+/* Navigation */
+nav {
+  background-color: #1e1e1e;
+  text-align: center;
+}
+
+nav li {
+  display: inline-block;
+  margin-left: -3px;
+}
+
+nav .first-item {
+  display: inline-block;
+  margin-left: 0px;
```

Any time the value is zero, you can leave off the units (`px` or `%`) because zero is always zero 👍 
```suggestion
  margin-left: 0;
```

I won't call this out every time you use `0px`, but consider making this change throughout your CSS.